package-lock.json

```diff
@@ -713,7 +713,7 @@
     "color-support": {
```

Comment:
Can you please revert the `package-lock.json` changes, rebase onto the current `master` and reinstall `semver` with the latest npm?